docs/user_guide/query_tool.md

```diff
@@ -50,33 +50,28 @@ Columns in the TSV are described below:
-After your query, click the `How to get data` button, then click the button in the pop-up window to download the TSV.
+A machine-optimized version of the query results, containing [URIs](https://www.ontotext.com/knowledgehub/fundamentals/linked-data-linked-open-data/) instead of descriptive labels for harmonized attributes and metadata of matching subjects, is also available for download as a TSV.
 
 Each row corresponds to a single matching subject session, except for [datasets configured to only return aggregate results](#protected-subject-level-results-for-aggregate-datasets).
 
 ??? abstract "Example query result TSV"
-    {{ read_table('./repos/neurobagel_examples/query-tool-results/cohort-participant-machine-results.tsv') }}
-
-Columns in the TSV are described below:
-* = required values
-
-| Column name | Description |
-| ---- | ---- |
-| DatasetName * | name of the dataset |
-| PortalURI | URL to a website or page about the dataset |
-| SubjectID * | subject label |
-| SessionID * | session label |
-| SessionFilePath | _(imaging sessions only)_ path to the session directory, or subject directory if only one session exists. Either an absolute path from the filesystem root where the dataset is stored, or a relative path from the dataset root for DataLad datasets. |
-| SessionType * | type of data acquired in the session, either `ImagingSession` or `PhenotypicSession`. Represents the nature of data being described, without denoting specific time or visits. e.g., A session in which both imaging and non-imaging data were acquired would be represented by separate rows, one per type. |
-| NumMatchingPhenotypicSessions * | _(subject-level)_ total number of phenotypic sessions for the subject which match the query |
-| NumMatchingImagingSessions * | _(subject-level)_ total number of imaging sessions for the subject which match the query |
-| SessionImagingModalities | _(imaging sessions only)_ imaging modalities acquired in the session, as URIs |
-| SessionCompletedPipelines | _(imaging sessions only)_ processing pipelines completed for the session, as URIs |
-| DatasetImagingModalities | _(dataset-level)_ imaging modalities acquired in at least one session in the dataset, as URIs |
-| DatasetPipelines | _(dataset-level)_ processing pipelines completed for at least one session in the dataset, as URIs |
+    {{ read_table('./repos/neurobagel_examples/query-tool-results/neurobagel-query-results-with-URIs.tsv') }}
+
+This file contains the same columns and data as the [descriptive query results TSV](#harmonized-tsv-data-with-descriptive-labels). 
+However, the harmonized terms in the following columns are provided in their raw URI form instead of as descriptive labels:
+
+| Column name |
+| ----- |
+| SessionType |
+| Sex |
+| Diagnosis |
+| Assessment |
+| SessionImagingModalities |
+| SessionCompletedPipelines |
+| DatasetImagingModalities |
+| DatasetPipelines |
```

Comment:
was this intentional to remove 'description'? I assume yes, but not sure where this information went?

Reply:
Only the names are included here because we're just listing the columns with URIs instead of descriptive labels. The descriptions are not provided again since they are exactly the same as in the table in the section `#### Harmonized TSV data with descriptive labels` (which is also linked right above this table).